### ISSUE TITLE
Missing form label

### ISSUE THREAD

**Issue by kamilfurtak** [bergben/ng2-file-input#34]
WAVE Web Accessibility Evaluation Tool show WCAG error.
Label should be explicity assciated with input element (https://www.w3.org/TR/WCAG20-TECHS/H44.html)

https://github.com/bergben/ng2-file-input/blob/3b3da27f4b86ef4920aa03f6eaa9d1846db41b5d/src/ng2-file-input.component.ts#L27
